index.html

```diff
@@ -19,29 +19,29 @@
             </nav>
         </header>
         <section class="container">
-            <h4 class="primary-text">
+            <h1 class="primary-text">
                 I’m Gabriel <br> Glad
                 to see you!
-            </h4>
+            </h1>
         </section>
 
         <section class="text-2">
```

Comment:
- [OPTIONAL] It would be better to not leave empty space in your file. This would give your project a professional look, semantically correct and follow html best practice. If you feel like having more  space in between lines in your code, you can increase that on vscode text editor or any text editor of your choice. See this [link](https://www.youtube.com/watch?v=PD_aNN29F44) for how to go about that.